website/docs/cloud-docs/api-docs/notification-configurations.mdx

```diff
@@ -112,6 +112,65 @@ For Slack and Microsoft Teams notifications, the payload conforms to the respect
 }
 ```
 
+### Workspace Notification Payload
+
+| Name                                         | Type   | Description                                                               |
+| -------------------------------------------- | ------ | ------------------------------------------------------------------------- |
+| `payload_version`                            | number | Currently always "2".                                                     |
+| `notification_configuration_id`              | string | ID of the configuration which defined this notification.                  |
```

Comment:
should it be "that triggered this notification" or that "created this notification". Defined feels weird to me as the user defined it. Thoughts?

```suggestion
| `notification_configuration_id`              | string | ID of the configuration that defined this notification.                  |
```

Reply:
(More language borrowed from the run payload -- will update that with whatever we decide).  I see what you mean, though neither "created" nor "triggered" feels write to me.  The notification configuration doesn't trigger the notification, it specifies the conditions that will trigger it.  "corresponding to"? "from which this notification results" lol? "caused"?  They all feel a little off to me; I don't have a strong opinion.